server/spec/consumer_resource_cdk_spec.rb

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+require 'spec_helper'
+require 'candlepin_scenarios'
+require 'rexml/document'
+
+describe 'Consumer CDK Resource' do
+
+  include CandlepinMethods
+
+  before(:each) do
+    @owner = create_owner random_string('test_owner')
+    @username = random_string("user")
+    @consumername = random_string("cdk_consumer")
+    @user = user_client(@owner, @username)
+    @cdk_product = create_product("cdk_product",
+                                  "CDK Product",
+                                  {:attributes => { :expired_after => "60"}})
+    @p_product1 = create_product("p_product_1",
+                                  "Provided Product 1")
+    @p_product2 = create_product("p_product",
+                                  "Provided Product 2")
+    @consumer = consumer_client(@user, @consumername, :system, 'cdk_user', facts= {:dev_sku => "cdk_product"})
+    installed = [
+        {'productId' => @cdk_product.id, 'productName' => @cdk_product.name},
```

Comment:
This product should not be an installed product -- it is the dev_sku marking product that will be looked up.